modules/op-resolver-bundle-config.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-bundles-config_{context}"]
+= Configuring the bundles resolver
+
+Optionally, you can change the default service account name and the default kind for pulling resources from a Tekton bundle.
+
+.Procedure
+
+. To edit the `TektonConfig` custom resource, enter the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
++
+. In the `TektonConfig` custom resource, specify configuration in the `pipeline.bundles-resolver-config` spec:
```

Comment:
"specify configuration" without any articles in front of it still reads off to me. I'd go with "the following" but I'm sure there are other alternatives that could work

Reply:
Agree